pages/desktop/regions/debug.py

```diff
@@ -12,10 +12,12 @@
 class Debug(Page):
 
     _region_select_locator = (By.NAME, 'region')
+    _region_option_locator = (By.CSS_SELECTOR, 'select[name="region"] option[value="%s"]')
 
     def __init__(self, testsetup):
         Page.__init__(self, testsetup)
         self.wait_for_element_visible(*self._region_select_locator)
 
     def select_region(self, region):
-        self.select_option(region, self._region_select_locator)
+        self.selenium.find_element(self._region_option_locator[0],
+                                   self._region_option_locator[1] % region).click()
```

Comment:
Can we just use `Select(self.selenium.find_element(*_region_select_locator)).select_by_value(region)` here? See [Select.select_by_value](http://selenium-python.readthedocs.org/en/latest/api.html?highlight=script#selenium.webdriver.support.select.Select.select_by_value).